install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -110,7 +110,14 @@ spec:
 
 {{- if .Values.nodeinit.reconfigureKubelet }}
                     echo "Changing kubelet configuration to --network-plugin=kubenet"
-                    sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
+                    if [[ -f /etc/default/kubelet ]];
+                    then
+                      sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
```

Comment:
not sure why we would need to escape the space character (i.e. `\ ` instead of simply ` `) here.